src/pip/_internal/utils/misc.py

```diff
@@ -128,7 +130,7 @@ def display_path(path):
     if possible."""
     path = os.path.normcase(os.path.abspath(path))
     if sys.version_info[0] == 2:
-        path = path.decode(sys.getfilesystemencoding(), 'replace')
+        path = fs_decode(path)
```

Comment:
You've lost the "replace" error handler here, so this has introduced the possibility of Unicode errors...